modules/sizeMappingV2.js

```diff
@@ -0,0 +1,464 @@
+      if (activeViewport[0] >= config.minViewPort[0]) {
+        if (activeViewport[1] >= config.minViewPort[1]) {
+          activeSizeBucket = config.minViewPort;
+        } else {
+          activeSizeBucket = [];
+        }
+      }
+    })
+  return activeSizeBucket;
+}
+
+function getRelevantMediaTypesForBidder(sizeConfig, activeViewport) {
+  if (checkBidderSizeConfigFormat(sizeConfig)) {
+    const activeSizeBucket = getActiveSizeBucket(sizeConfig, activeViewport);
+    return sizeConfig.filter(config => config.minViewPort === activeSizeBucket)[0]['relevantMediaTypes'];
+  }
+  return [];
+}
+
+function getBids({ bidderCode, auctionId, bidderRequestId, adUnits, labels, src }) {
+  return adUnits.reduce((result, adUnit) => {
+    if (isLabelActivated(adUnit, labels, adUnit.code)) {
+      if (adUnit.mediaTypes && isValidMediaTypes(adUnit.mediaTypes)) {
+        const { mediaTypes, activeSizeBucket, activeViewport, transformedMediaTypes } = getFilteredMediaTypes(adUnit.mediaTypes);
+        logInfo(`SizeMappingV2:: AdUnit: ${adUnit.code}, Bidder: ${bidderCode} - Active size buckets after filtration: `, activeSizeBucket);
+        logInfo(`SizeMappingV2:: AdUnit: ${adUnit.code}, Bidder: ${bidderCode} - Transformed mediaTypes after filtration: `, transformedMediaTypes);
+        logInfo(`SizeMappingV2:: AdUnit: ${adUnit.code}, Bidder: ${bidderCode} - mediaTypes that got filtered out: `, Object.keys(mediaTypes).filter(mt => Object.keys(transformedMediaTypes).indexOf(mt) === -1));
+
+        // check if adUnit has any active media types remaining, if not drop the adUnit from auction,
+        // else proceed to evaluate the bids object.
+        if (Object.keys(transformedMediaTypes).length === 0) {
+          logInfo(`SizeMappingV2:: Ad Unit: ${adUnit.code} is disabled since there are no active media types after sizeConfig filtration.`);
+          return result;
+        }
+        result
+          .push(adUnit.bids.filter(bid => bid.bidder === bidderCode)
+            .reduce((bids, bid) => {
+              if (isLabelActivated(bid, labels, adUnit.code)) {
+                // handle native params
+                const nativeParams = adUnit.nativeParams || deepAccess(adUnit, 'mediaTypes.native');
+                if (nativeParams) {
+                  bid = Object.assign({}, bid, {
+                    nativeParams: processNativeAdUnitParams(nativeParams)
+                  });
+                }
+
+                bid = Object.assign({}, bid, getDefinedParams(adUnit, ['mediaType', 'renderer']));
+
+                if (bid.sizeConfig) {
+                  const relevantMediaTypes = getRelevantMediaTypesForBidder(bid.sizeConfig, activeViewport);
+                  if (relevantMediaTypes.length === 0) {
+                    logError(`SizeMappingV2:: Bidder: ${bid.bidder} in Ad Unit: ${adUnit.code} has not configured sizeConfig property correctly. This bidder won't be eligible for sizeConfig checks and will remain active.`);
```

Comment:
This may be a minor thing, but could you flip the order of the adUnit and bidder in these log messages (and below this point)?  

The majority of the messages earlier in the code use adUnit > bidder type of listing, so the switch here to bidder > adUnit is a bit distracting when looking at all the console messages together in the browser.

Reply:
Made the change. Please have a look.